src/Microsoft.Extensions.Logging.Console/ConsoleLogScope.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+#if NET451
+using System.Runtime.Remoting;
+using System.Runtime.Remoting.Messaging;
+#else
+using System.Threading;
+#endif
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    public class ConsoleLogScope
+    {
+        private readonly string _name;
+        private readonly object _state;
+
+        public ConsoleLogScope(string name, object state)
+        {
+            _name = name;
+            _state = state;
+        }
+
+        public ConsoleLogScope Parent { get; private set; }
```

Comment:
Probably no as the details of this relationship is encapsulated within this class and the ConsoleLogger class need not know about it